### PR TITLE
Fix links in the yaml used

### PR DESCRIPTION
**Pull request by piyush-garg**
This will update all the yaml links to point
to release-tech-preview-1 branch and use
release-tech-preview-1 branch for frontend
and backend repo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -650,7 +655,7 @@ $ oc expose svc el-vote-app
 
 ## Configuring GitHub WebHooks
 
-Now we need to configure webhook-url on [backend](https://github.com/openshift-pipelines/vote-api) and [frontend](https://github.com/openshift-pipelines/vote-ui) source code repositories with the Route we exposed in the previously.
+Now we need to configure webhook-url on [backend](https://github.com/openshift-pipelines/vote-api/tree/release-tech-preview-1) and [frontend](https://github.com/openshift-pipelines/vote-ui/tree/release-tech-preview-1) source code repositories with the Route we exposed in the previously.
```
Comment (pthangad) [inline README.md:658]: Please do change (we exposed in the previously -> we exposed previously)

**Review by pthangad — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (piyush-garg) [reply README.md:658]: is it not better we do this in master?

Comment (vdemeester) [reply README.md:658]: both ? :stuck_out_tongue_closed_eyes:

**Review by vdemeester — APPROVED** [review]
/lgtm

**Pull request merged**